examples/certified-counter/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "certification_frontend",
+  "version": "0.1.0",
+  "description": "Internet Computer starter application",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "JavaScript",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "webpack",
+    "prebuild": "npm run generate",
+    "start": "webpack serve --mode development --env development",
+    "prestart": "npm run generate",
+    "generate": "dfx generate backend"
+  },
+  "dependencies": {
+    "@dfinity/certificate-verification": "workspace:*"
```

Comment:
Since this `package.json` is using the `pnpm` format with the dependencies version we should also use `pnpm` on the scripts section imo